serverAddons/PRA3/addons/PRA3_Server/Revive/Treatments/fn_clientInitTreatments.sqf

```diff
@@ -0,0 +1,148 @@
+#include "macros.hpp"
+/*
+    Project Reality ArmA 3
+
+    Author: BadGuy, joko // Jonas, NetFusion
+
+    Description:
+    Handles bandages.
+
+    Parameter(s):
+    -
+
+    Returns:
+    None
+*/
+
+/*
+ * WEAPON SWITCHING
+ */
+[QGVAR(SwitchWeapon), {
+    params ["_item"];
+
+    // Restore old action
+    [QGVAR(RestoreWeapon)] call CFUNC(localEvent);
+
+    // Move the weapon on back
+    PRA3_Player action ["SwitchWeapon", PRA3_Player, PRA3_Player, 99];
+
+    // Create a weapon holder and fill it with the dummy item
+    private _weaponHolder = createVehicle ["GroundWeaponHolder", [0, 0, 0], [], 0, "NONE"];
```

Comment:
It's a new bi command, to create only a 3D object: https://community.bistudio.com/wiki/createSimpleObject